enterprise/frontend/src/metabase-enterprise/audit_app/pages/AuditQueryDetail.jsx

```diff
@@ -25,7 +25,7 @@ const AuditQueryDetail = ({ params: { queryHash } }) => (
 
       return (
         <AuditContent
-          title="Query"
+          title={t`Query`}
```

Comment:
There are multiple ways to come to this page, but one of the ways to get here is to:
1. Go to Admin > Audit > Databases > All Databases
2. Select any database
3. Click on any `ad-hoc` query

![Screenshot 2023-08-15 at 5 03 03 PM](https://github.com/metabase/metabase/assets/1937582/4f9eb05f-03c9-4410-84b7-bf6fb3ebd45a)